src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/ScopeTab.tsx

```diff
@@ -0,0 +1,36 @@
+import { WbsElementStatus, WorkPackage, isGuest } from 'shared';
+import CheckList from '../../../components/CheckList';
+import { useCurrentUser } from '../../../hooks/users.hooks';
+
+const ScopeTab = ({ workPackage }: { workPackage: WorkPackage }) => {
+  const user = useCurrentUser();
+
+  const checkListDisabled = workPackage.status !== WbsElementStatus.Active || isGuest(user.role);
+
+  return (
+    <>
+      <CheckList
+        title={'Expected Activities'}
+        items={workPackage.expectedActivities
+          .filter((ea) => !ea.dateDeleted)
+          .map((ea) => {
+            return { ...ea, resolved: !!ea.userChecked, user: ea.userChecked, dateAdded: ea.dateAdded };
+          })}
+        isDisabled={checkListDisabled}
+      />
+      <div style={{ marginTop: 5 }}>
```

Comment:
por que div

Reply:
this is actually what divs are usually used for fyi, when you wanna apply styling around a component, I can make it a Box tho for consistency sake